cli/tests/pcluster/config/test_section_efs.py

```diff
@@ -173,7 +173,11 @@ def test_efs_section_to_cfn(mocker, section_dict, expected_cfn_params):
             utils.merge_dicts(
                 DefaultCfnParams["cluster"].value,
                 DefaultCfnParams["efs"].value,
-                {"MasterSubnetId": "subnet-12345678", "AvailabilityZone": "mocked_avail_zone"},
+                {
+                    "MasterSubnetId": "subnet-12345678",
+                    "AvailabilityZone": "mocked_avail_zone",
+                    "EC2IAMPolicies": "arn:aws:iam::aws:policy/CloudWatchAgentServerPolicy",
```

Comment:
You could add the `CloudWatchAgentServerPolicy` to the `DefaultCfnParams["cluster"]` since by default it is included in the EC2IAMPolicies